src/styles/main.scss

```diff
@@ -0,0 +1,17 @@
+@import "utils/keyframes";
+@import "utils/variables";
+@import "blocks/clock.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
```

Comment:
`justify-content: center;` and `align-content: center;` are redundant, remove them